### ISSUE TITLE
Error higher after refinement of labels

### ISSUE THREAD

**Issue by cdj5044** [DeepLabCut/DeepLabCut#911]
Hello, 

I first want to thank you for all the work you've done on this application; it is extremely useful and packaged really nicely. 

Here's a run-down of the issue I'm having:

- I trained a model using the resnet-101 base model, for videos with human subjects, using 250,000 iterations and 1,000 training frames. The evaluation of this model gave a train error of 2.59 and test error of 4.43 pixels. 

- I analyzed 8 videos with the model, extracted 20 outlier frames from each, using low certainty as the parameter, and corrected the markers for these 160 frames. 

- I appended this data to the original training set and retrained the model using 300,000 iterations. The evaluation of this model gave a train error of 3.02 and test error of 6.79 pixels (actually at the 275,000 iteration checkpoint, which had the best performance). Also, the loss did plateau for the last 40,000 or so iterations: 0.00238, 0.00233, 0.00237, 0.00227. 

Is there some reason why the error would increase after refining the labels? Should I try a different method of extracting outlier frames, such as the jump method, and retrain the model again?

Thank you in advance for your help with this issue!

Best Wishes,
Caleb

**Comment by MMathisLab** [comment]
Hi Caleb,

Did you refrain from scratch? If you only added a few frames and train on top, it can not be worse, but just a different split of the data. I would recommend looking at video performance and seeing if it looks better. Pixel error is but just one metric.

Also, you should be mindful that with ResNet-101 you should make changes to the pose_cfg.yaml before training and set intermediate supervision to true. We really don’t recommend 101 unless you really need it (complex scenes, etc). 

See more here: https://github.com/DeepLabCut/DeepLabCut/wiki/What-neural-network-should-I-use%3F

**Comment by cdj5044** [comment]
Hi Mackenzie, 

Thank you so much for your response and all the helpful info! I am analyzing the videos to look at video performance right now, as you suggested. 

Related to using the ResNet-101, we only selected this ResNet based on the recommendation to use it for videos involving human subjects, at this documentation page: https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/functionDetails.md#a-create-a-new-project. So thank you for forwarding the documentation specific to selecting the right neural network. 

I did actually train a model using ResNet-50 prior to switching to -101, and at least the pixel error was much worse: train/test errors of about 4 and 11 respectively. 

Thanks!
Caleb
